gherkin/java/src/main/groovy/io/cucumber/gherkin/GherkinDialects.gsp

```diff
@@ -0,0 +1,32 @@
+package io.cucumber.gherkin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.unmodifiableList;
+
+final class GherkinDialects {
+    static final Map<String, GherkinDialect> DIALECTS = new HashMap<>();
```

Comment:
Consider a `LinkedHashSet` to preserve the order from `gherkin-languages.json`. This may be useful for `GherkinDialectProvider.getLanguages()`

Reply:
Ok